### PR TITLE
modules/main/extra-modules.conf: enable dm_verity and overlay modules

### PR DESCRIPTION
**Pull request by sespiros**
This will allow mounting partitions with dm-verity in the initrd as well as overlayfs mounts.

### CODE REVIEW

File: modules/main/extra-modules.conf [hunk 1]
```diff
@@ -18,6 +18,8 @@ algif_skcipher
 cryptomgr
 dm_crypt
 dm_mod
+dm_verity
+overlay
```
Comment (alfonsosanchezbeato) [inline modules/main/extra-modules.conf:22]: Could you please provide some more information on why we need this?  Is this expected to be used only for the CVM use case?

Comment (sespiros) [reply modules/main/extra-modules.conf:22]: Yes, the CVM use case is the driver for this. The change is described in spec FO164.

**Review by alfonsosanchezbeato — COMMENTED** [review]
I think this is fine, but please add to `modules/server/extra-modules.conf` instead so we get this for non-canonical kernels.

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks

**Pull request merged**